### DISCUSSION TITLE
Do not require a comment when "Request Changes" is selected, if you have justed added inline comments

### DISCUSSION THREAD

**Discussion by TheJaredWilcurt** [community/community#76808] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

If you have added inline comments, there is an extremely high liklihood that it is to request someone change something. So to then require me to type the same thing *again* that I already typed in a context specific area into a box that will put it in a non-context specific area is very annoying. My team at work just straight up stopped using this built in tool because it does not work well, and replaced it with a custom label.

**Solution:**

If the user requesting changes has added any inline comments since the last time a request changes/approval was made, then they should not be required to type anything into the approval/request changes text box.

Honestly, even if they didn't add any inline comments, but *someone else did*, they should still be able to set the PR status to "Changes Requested" without a useless comment ('Do what he said', 'see: inline comments', or anything else equally useless and time wasting).

**Comment by ol1s** [comment]
Yeah totally agree

**Comment by stwlam** [comment]
Is there any indication of whether or not this request will be acted on? The OP lays out a crucial flaw of the feature that makes it unusable in many cases.

**Comment by Yuniac** [comment]
Will this ever be implemented? 

It's rather annoying that you write a couple of comments describing what's wrong in a PR or what needs changing then you hit `Request Changes` just for it to fail because you... didn't leave a comment describing what's need changing even though you technically did

**Comment by mblakesley** [comment]
Simple QoL feature. I've wanted this since I started using Github. Ofc it's easy enough to work around, but it's annoying to everyone.

**Comment by realmocaccino** [comment]
Up
